pkg/edgecache/manager.go

```diff
@@ -0,0 +1,192 @@
+package edgecache
+
+import (
+	"context"
+	"path/filepath"
+	"time"
+
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+	"k8s.io/klog/v2"
+	"sigs.k8s.io/blob-csi-driver/pkg/edgecache/blob_cache_volume"
+	"sigs.k8s.io/blob-csi-driver/pkg/edgecache/cache_volume_service"
+	"sigs.k8s.io/blob-csi-driver/pkg/edgecache/csi_mounts"
+)
+
+type Manager struct {
+	connectTimeout int
+	configEndpoint string
+	mountEndpoint  string
+}
+
+func NewManager(connectTimeout int, configEndpoint string, mountEndpoint string) *Manager {
+	return &Manager{
+		connectTimeout: connectTimeout,
+		configEndpoint: configEndpoint,
+		mountEndpoint:  mountEndpoint,
+	}
+}
+
+func GetStagingPath(path string) string {
+	return filepath.Join(path, "edgecache")
+}
+
+func sendGetVolume(client cache_volume_service.CacheVolumeClient, account string, container string) (error, bool) {
```

Comment:
cosmetic: is there significant to the GetStagingPath vs sendGetVolume from a first-letter-capitalized perspective?

Reply:
In go, 'public' functions in a package are capitalized. Internal functions are not, so the interface functions in this package use capital first letters.